sql/api/src/main/scala/org/apache/spark/sql/types/StringType.scala

```diff
@@ -25,7 +25,7 @@ import org.apache.spark.annotation.Stable
  * @since 1.3.0
  */
 @Stable
-class StringType private() extends AtomicType {
+class StringType private(val collationId: Int) extends AtomicType {
```

Comment:
Could you add `@param` above and describe `collationId`, please.

Reply:
Done, thanks.